libs/dataviews-client/src/resolve-dataviews.ts

```diff
@@ -170,6 +170,17 @@ export const resolveResourcesFromDatasetConfigs = (
     })
 }
 
+export const generateDataviewDatasetResourceKey = (
```

Comment:
Using a key to replace the previous downloaded geojson so we don't need to worry about spamming the store